templates/base.html

```diff
@@ -11,7 +11,6 @@
     <script src="{% static 'js/jquery-ui.min.js' %}"></script>
     <script src="{% static 'js/bootstrap.min.js' %}"></script>
     <script src="{% static "js/jquery.dataTables.min.js" %}" type="text/javascript"></script>
-    <script src="{% static "js/superlink.jquery.js" %}" type="text/javascript"></script>
```

Comment:
It's MIT licensed: https://github.com/osahan/superlink/blob/master/LICENSE

However, it's not actually used anywhere so out it goes